src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -240,7 +240,7 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
      * Returns a value indicating whether the current asset is taggable
      */
     private isTaggableAssetType = (asset: IAsset): boolean => {
-        return asset.type === AssetType.Image || asset.type === AssetType.VideoFrame;
+        return asset.type !== AssetType.Unknown && asset.type !== AssetType.Video;
```

Comment:
Is an asset with AssetType === Video not considered taggable?

Reply:
That's right. Video is the parent but the real taggable assets are his child of type VideoFrame